src/pages/PrivacyPolicy/index.js

```diff
@@ -15,12 +15,22 @@ function PrivacyPolicy({ translation }) {
     }
   ];
 
+  function WebOrMobileClassName() {
+    var className = '';
+
+    if (window.innerWidth <= 500) {
+      className = 'container-mobile';
+    }
+
+    return className;
+  }
+
   return (
     <PageLayout
       title={translation('PrivacyPolicyPage-PageName')}
       descriptionParagraphs={[translation('PrivacyPolicyPage-Description')]}
       breadcrumbsData={breadcrumbs}>
-      <Container fluid="md">
+      <Container fluid="md" className={WebOrMobileClassName()}>
```

Comment:
Tou a ver o preview e continuo a ver o corte 🤔 
Assim a primeira revisão do código, não percebo o que pode ser 🤔 

https://ardc-website-git-fork-catarina-pinto-fix-la-de9c12-ardc-website.vercel.app/privacy

<img width="917" alt="image" src="https://github.com/As-Raparigas-do-Codigo/ardc-website/assets/11148726/22e81952-be08-4872-996d-e7c7704653ae">

Reply:
![Screenshot_20240208_215150_com huawei browser](https://github.com/As-Raparigas-do-Codigo/ardc-website/assets/81467625/9563ce01-4c59-47f5-9f16-dbdbd9f7e4ca)

Ao abrir o preview no telemóvel aparece-me assim. No browser no pc já aparece também corrigido mas ao início aparecia cortado também

Reply:
Testei agora no computador e acontece-me o mesmo: quando mudo para mobile fica sobreposto. Fazendo refresh da página, no entanto, passa a aparecer corretamente. No telemóvel, por sua vez, aparece imediatamente corrigido

Reply:
Hum interessante 🤔 Não sei o que pode ser 😅 
Se calhar não podemos confiar a 100% no simulador do chrome. Vou tagar outros mentores

Reply:
Pode ser simplesmente a ordem de renderização/velocidade da internet